vuu-ui/packages/vuu-utils/src/filters/filter-utils.ts

```diff
@@ -182,3 +186,134 @@ export class FilterAggregator extends EventEmitter<FilterEvents> {
     }
   }
 }
+
+export type ColumnFilterStoreEvents = {
+  onAdd: (filter: VuuFilter) => void;
```

Comment:
as a user of this, will we care whether a filter has been added or removed ? 
Won't we just want to know that the overall filter represented by the whole colelction has changed ?